generated/nidmm/session.py

```diff
@@ -569,6 +581,9 @@ def __setattr__(self, key, value):
             raise AttributeError("'{0}' object has no attribute '{1}'".format(type(self).__name__, key))
         object.__setattr__(self, key, value)
 
+    def lock(self):  # TODO(texasaggie97) Need to figure out how to document this
```

Comment:
If it doesn't go in the PR, please open an issue.

Reply:
Figured out.